cache-config/testing/ort-tests/t3c_mode_test.go

```diff
@@ -15,18 +16,19 @@ package orttest
 */
 
 import (
-	"fmt"
-	"github.com/apache/trafficcontrol/cache-config/testing/ort-tests/tcdata"
-	"github.com/apache/trafficcontrol/cache-config/testing/ort-tests/util"
 	"os"
 	"os/user"
+	"path/filepath"
 	"strconv"
 	"syscall"
 	"testing"
 	"time"
+
+	"github.com/apache/trafficcontrol/cache-config/testing/ort-tests/tcdata"
+	"github.com/apache/trafficcontrol/cache-config/testing/ort-tests/util"
 )
 
-const TrafficServerOwner = "ats"
+const trafficServerOwner = "ats"
```

Comment:
Is there a big reason to lowercase this? It doesn't strictly need to be accessible outside the library, but the capitalization makes it more obvious inside tests that the variable is "global" and not local wherever it's being used.

Reply:
I just didn't think it needed to be exported, and didn't want to write a GoDoc comment for it. But you actually can't export from tests so there's no real distinction. It doesn't need to be changed, but I am moving the definition into defaults_test.go with the cache host name. So I'll probably also add a comment, in total reversal of the motivations for this change.